README.md

```diff
@@ -854,6 +854,43 @@
         .call(tron.led);
     ```
 
+  - Leave a blank line after blocks and before the next statement
+
+    ```javascript
+    // bad
+    if (true) {
+      var a = 2;
+    }
+    var b = 3;
+
+    // good
+    if (true) {
+      var a = 2;
```

Comment:
Agreed on it being pretty minor. I implemented it for JSCS and it will be landing in the next version which should help to reason about the rule as well.

You can view the tests for the rule here:
https://github.com/TheSavior/node-jscs/blob/require-padding-newlines-after-block-declarations/test/rules/require-padding-newlines-after-blocks.js